internal/http/services/owncloud/ocdav/delete.go

```diff
@@ -67,6 +67,15 @@ func (s *svc) handleDelete(ctx context.Context, w http.ResponseWriter, r *http.R
 
 	req := &provider.DeleteRequest{Ref: ref}
 
+	// do not allow deleting spaces via dav endpoint - use graph one instead
+	// we get a relative reference coming from the space root
+	// so if the path is "empty" we a referencing the space
+	if ref.Path == "." {
```

Comment:
That could also be node further down in the tree. 

IMO a better check would be if storageID and opaqueID are the same.

Reply:
That is only working for decomposedfs. Eos for example is having different IDs for space and opaque. We need are more sophisticated way to determine a space root from other items.

For now it is fine though, the `ref` is created via `makeRelativeReference` which uses the space root and a path from there.